resources/js/actions/types.js

```diff
@@ -9,6 +12,19 @@ export const VALIDATE_LNAME = "VALIDATE_LNAME";
 export const FRESH_STATE = "FRESH_STATE";
 export const SHOW_PASSWORD = "SHOW_PASSWORD";
 export const SHOW_CONFIRMPASSWORD = "SHOW_CONFIRMPASSWORD";
-export const TOGGLE_SUBMITBTN = "TOGGLE_SUBMITBTN"
+export const TOGGLE_SUBMITBTN = "TOGGLE_SUBMITBTN";
 export const USER_AUTH_DETAILS = "USER_AUTH_DETAILS";
-
+/*
+    Category actions
+*/
```

Comment:
you can create a different folder for authentication actions and category actions and other actions so that you dont need to depend on comments to categorize them...